inc/Engine/Optimization/RUCSS/Admin/Settings.php

```diff
@@ -481,15 +481,12 @@ public function display_wrong_license_notice() {
 			return;
 		}
 
-		$boxes = get_user_meta( get_current_user_id(), 'rocket_boxes', true );
-
-		if ( in_array( 'rucss_wrong_licence_notice', (array) $boxes, true ) ) {
-			return;
-		}
+		$main_message = __( "We couldn't generate the used CSS because you're using a nulled version of WP Rocket. You need an active license to use the Remove Unused CSS feature and further improve your website's performance..", 'rocket' );
```

Comment:
```suggestion
		$main_message = __( "We couldn't generate the used CSS because you're using a nulled version of WP Rocket. You need an active license to use the Remove Unused CSS feature and further improve your website's performance.", 'rocket' );
```